controllers/managedocs_controller.go

```diff
@@ -1086,3 +1118,66 @@ func getCSVByPrefix(csvList opv1a1.ClusterServiceVersionList, name string) *opv1
 	}
 	return csv
 }
+
+func (r *ManagedOCSReconciler) reconcileK8sMetricsServiceMonitorAuthSecret() error {
```

Comment:
We use uppercase for abbreviations (unless they appear at the start of a variable or function name):

```suggestion
func (r *ManagedOCSReconciler) reconcileK8SMetricsServiceMonitorAuthSecret() error {
```